extensions/wikia/Chat2/ChatBanListSpecial_helper.php

```diff
@@ -197,21 +197,20 @@ public function loadData() {
 	private function getUserLinks( $user ) {
 
 		$userIsBlocked = $this->wg->User->isBlocked( true, false );
-		$oEncUserName = urlencode( $user->getName() );
+		$username = $user->getName();
 		$links = [
 			0 => "",
 			1 => Linker::link(
-				Title::newFromText( 'Contributions', NS_SPECIAL ),
-				$this->wg->Lang->ucfirst( wfMsg( 'contribslink' ) ),
-				[ 'target' => $oEncUserName]
+				SpecialPage::getSafeTitleFor( 'Contributions', $username ),
+				$this->wg->Lang->ucfirst( wfMsg( 'contribslink' ) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)